### ISSUE TITLE
Does not start due to error finding './../actions/commands/actions' module on Linux on ChromeOS

### ISSUE THREAD

**Issue by nikki93** [VSCodeVim/Vim#4341]
**Describe the bug**
I'm trying to run VSCode by installing it on Linux on ChromeOS. I don't think the ChromeOS part should change much / it should work as a normal Linux environment. I've enabled the Vim extension in it but it just does not start up--cursor is the normal cursor and I don't enter Vim normal mode. Here's an error stack trace:
```Activating extension 'vscodevim.vim' failed: Cannot find module './../actions/commands/actions'
Require stack:
- /home/snikhilesh/.vscode/extensions/vscodevim.vim-1.12.0/out/src/register/register.js
- /home/snikhilesh/.vscode/extensions/vscodevim.vim-1.12.0/out/src/actions/operator.js
- /home/snikhilesh/.vscode/extensions/vscodevim.vim-1.12.0/out/src/actions/include-all.js
- /home/snikhilesh/.vscode/extensions/vscodevim.vim-1.12.0/out/extension.js
- /usr/share/code/resources/app/out/vs/loader.js
- /usr/share/code/resources/app/out/bootstrap-amd.js
- /usr/share/code/resources/app/out/bootstrap-fork.js.
```

**To Reproduce**
1. Enable 'Linux apps' on ChromeOS, install VSCode through the .deb. I don't know if the fact that it's on ChromeOS is relevant in this case, but it might be.
2. Install the Vim extension from the extension sidebar in VSCode.
3. Restart VSCode just in case.
4. No Vim mode. :(

**Expected behavior**
Enter Vim mode.

**Screenshots**
![image](https://user-images.githubusercontent.com/49151/70410717-d7be8480-1a04-11ea-8cdd-0cc639d961ff.png)

- Extension (VsCodeVim) version: 1.12.0
- VSCode version: 1.40.2
- OS: Linux on ChromeOS

**Comment by BolotskyDev** [comment]
Same on Mac, resolved by installing 1.11.3 (previous) version

**Comment by nikki93** [comment]
@BolotskyDev -- Yep, that fixed it for me. Guess this is a general issue with the latest version!

**Comment by CermakM** [comment]
It is, the file is missing in `~/.vscode/extensions/vscodevim.vim-1.12.0/out/src/actions/commands/`

**Comment by BolotskyDev** [comment]
@CermakM yep, I have it checked as well. It seems like a very popular problem with latest 1.12 patch , here is another related issue #4342

**Comment by rebornix** [comment]
Fixed in 1.12.1, sorry for the trouble.
